cli/dxapplet/src/code.sh

```diff
@@ -36,6 +51,7 @@ main() {
     fi
 
     # load gVCFs
+    # initialize and load database
```

Comment:
did this comment come back from a rebase? (at some point the steps to initialize and load the database got split up)